.github/workflows/perf-check.yml

```diff
@@ -0,0 +1,36 @@
+name: Perf check
```

Comment:
The target in the makefile is `perf-compare`, so how about substituting "compare" for "check" everywhere?

Reply:
@nwt: Yes, that's fine. I've made that change. Once again, I wasn't being arbitrary in my original wording: Saying "check" was my way of communicating that comparing the performance between runs is _not_ the goal of this workflow. But it's subtle and not worth obsessing over.